mp/src/vgui2/vgui_controls/CvarComboBox.cpp

```diff
@@ -20,6 +20,18 @@ CvarComboBox::CvarComboBox(Panel *parent, const char *panelName, int numLines, b
 {
     InitSettings();
     m_bIgnoreMissingCvar = ignoreMissingCvar;
+    float m_flCvarMin, m_flCvarMax;
+    m_iCvarMin = m_cvar.GetMin(m_flCvarMin) ? RoundFloatToInt(m_flCvarMin) : 0;
+    m_iCvarMax = m_cvar.GetMin(m_flCvarMax) ? RoundFloatToInt(m_flCvarMax) : 0;
+    float m_flCvarMin;
```

Comment:
Only use `m_` for member variables. You also seem to be declaring `m_flCvarMin` twice here.

Reply:
Woops! All this should do is needlessly call `Reset()` anyway. Will fix this up.